features/activemq/component/pom.xml

```diff
@@ -0,0 +1,31 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.opennms.features</groupId>
+    <artifactId>org.opennms.features.activemq</artifactId>
+    <version>19.0.0-SNAPSHOT</version>
+  </parent>
+  <packaging>bundle</packaging>
+  <groupId>org.opennms.features.activemq</groupId>
+  <artifactId>component</artifactId>
```

Comment:
Can you change this artifactId to "org.opennms.features.activemq.component"? That will make the JAR name easier to identify (since it would be difficult to tell what "component-19.0.0.jar" contains just by looking at its name).